runtime/vm/jvmfree.c

```diff
@@ -267,6 +267,13 @@ deallocateVMThread(J9VMThread * vmThread, UDATA decrementZombieCount, UDATA send
 	}
 #endif
 
+#if JAVA_SPEC_VERSION >= 16
+	if (NULL != vmThread->ffiArgs) {
```

Comment:
Free of NULL is allowed, so you could remove this check (I believe the expected case would be that it's not NULL).

Reply:
Removed the NULL check as suggested above.